qcore/geo.py

```diff
@@ -17,35 +17,12 @@ class InputError(Exception):
     pass
 
 
-def get_distances(locations: np.ndarray, lon: float, lat: float):
+def get_distances(locations: np.ndarray, lon: np.float, lat: np.float):
```

Comment:
Docstring needs to be updated slightly since the function now supports multiple reference locations
And the type hints for lon & lat should be `Union[float, np.ndarray]`
Also I think in general its better to use float than np.float